doc/conf.py

```diff
@@ -26,18 +26,18 @@
 
 def _workaround_imports_typechecking():
     """Load some packages needed for type annotations."""
-    import ipywidgets
-    import pandas as _
-    import xarray
+    import ipywidgets  # noqa
+    import pandas  # noqa
+    import xarray  # noqa
 
 
 def _prevent_sphinx_circular_imports_bug():
     # https://github.com/sphinx-doc/sphinx/issues/9243
-    import sphinx.builders.html
-    import sphinx.builders.latex
-    import sphinx.builders.texinfo
-    import sphinx.builders.text
-    import sphinx.ext.autodoc
+    import sphinx.builders.html  # noqa
+    import sphinx.builders.latex  # noqa
+    import sphinx.builders.texinfo  # noqa
+    import sphinx.builders.text  # noqa
+    import sphinx.ext.autodoc  # noqa
```

Comment:
just curious: Which warnings are suppressed here?

Reply:
unused imports, the new pre-commit tool would remove them without the noqa. Then sphinx would break.